docs/site/Decorators_repository.md

```diff
@@ -137,13 +137,128 @@ added here as this feature progresses._
 
 Syntax:
 
-- `@belongsTo`
-- `@hasOne`
-- `@hasMany`
+- `@belongsTo(targetResolver: EntityResolver<T>, definition?: Partial<BelongsToDefinition>)`
+  - Many-to-one connection between models.
+  - A `Todo` belongs to a `TodoList`.
+  - See [here](BelongsTo-relation.md) for more details.
+
+{% include code-caption.html content="todo.model.ts" %}
+
+```
+import {belongsTo} from '@loopback/repository';
+import {TodoList} from './todo-list.model';
+
+export class Todo extends Entity {
+  // properties
+
+  @belongsTo(() => TodoList)
+  todoListId: number;
+
+  // etc
+}
+```
+
+- `@hasOne(targetResolver: EntityResolver<T>, definition?: Partial<HasOneDefinition>)`
+  - One-to-one connection between models.
+  - A `TodoList` has one `image`.
+  - See [here](hasOne-relation.md) for more details.
```

Comment:
Let's use a more descriptive link title please, e.g.
```suggestion
  - See [HasOne relation](hasOne-relation.md) for more details.
```

The same comment applies to other similar links below.